ExtendedColorSchemes/Plugin.cs

```diff
@@ -37,7 +37,7 @@ public void OnApplicationStart()
         [OnExit]
         public void OnApplicationQuit()
         {
-
+            _harmony.UnpatchSelf();
```

Comment:
There's a reason why I haven't unpatched my stuff, this method is called too early, before the game actually calls `Save`, so it will potentially break the save data by saving the custom color schemes.